### ISSUE TITLE
Export Report to HTML or other web browser support file type

### ISSUE THREAD

**Issue by jbuettner00** [microsoft/AttackSurfaceAnalyzer#697]
It would be nice to be able export the analysis report to an HTML file, or other browser format, so that it can be passed to anyone and they can open it in any web browser and see the results without having to look through the json file or create a tool to parse the json file or have access to the original instance of the ASA where the report was created.

**Option to select to export ASA report to an HTML**
A clear and concise description of what you want to happen. E.g. `I want to be able to run a command to see what runs are in the database.`

**Describe alternatives you've considered**
I have thought of a work around or another way to do this but they don't solve all my needs.
`I understand that I can view the database in sqlite but it would be more convenient to have this feature.`

**System Configuration (please complete the following information):**
* OS: Windows 11
* OS Version: Windows 11 Enterprise version 21H2
* Application Version: 2.3.299
* CLI or GUI: both

**Additional context**
N/A

**Comment by jbuettner00** [comment]
Hmm...I guess this can actually be handled by the web browser's Save As feature on the Report page of the MS ASA. It creates an Attack Surface Analyzer.html file and an associated Attack Surface Analyzer_files folder with the necessary data to re-open the report in any web browser. I think this will suffice for what I need.

**Comment by gfs** [comment]
ASA supports exporting the report as Sarif which has a number of view clients available (including the sarif web component: https://microsoft.github.io/sarif-web-component/) 

Have you tried with the sarif format report?

**Comment by jbuettner00** [comment]
I have not tried the sarif export, I wasn't aware of that. So is it only available from the command line?

**Comment by gfs** [comment]
Yes, I believe you can only export sarif from the command line currently. I can take a look at adding a sarif export button for the GUI as well. Thanks for the feedback.

**Comment by gfs** [comment]
Keeping this issue open to track potentially adding a sarif export button to the GUI.
